### PR TITLE
Update Ecuadorian tax rates

### PR DESCRIPTION
**Pull request by LouiseBonnard**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Update localization packs data, update tax rates - following the Avalara information [here](https://www.avalara.com/vatlive/en/vat-rates/international-vat-and-gst-rates.html)
| Type?         | improvement
| Category?     | LO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | It is part of this [EPIC](https://github.com/PrestaShop/PrestaShop/issues/15829).

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16181)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]
lgtm

**Comment by sarahdib** [comment]
@LouiseBonnard there is a problem when we try to upgrade the pack Ecuadorian and when we install PS with Ecuadorian country there is an error in the BO

Capture d’écran 2019-10-30 à 11 08 40
Capture d’écran 2019-10-30 à 11 18 52

File: localization/ec.xml [hunk 1]
```diff
@@ -8,9 +8,13 @@
 	</languages>
 	<taxes>
 		<tax id="1" name="IVA EC 12%" rate="12" />
+		<tax id="2" name="IVA EC 0%" rate="0" />
 		
 		<taxRulesGroup name="EC Standard Rate (12%)">
 			<taxRule iso_code_country="ec" id_tax="1" />
+
```
Comment (eternoendless) [inline localization/ec.xml]: Missing closing tag:

```suggestion
		</taxRulesGroup>
```

**Review by eternoendless — CHANGES_REQUESTED** [review]

**Review by eternoendless — APPROVED** [review]
Looks good now

**Pull request merged**